llvm/include/llvm/CodeGen/MachineBranchProbabilityInfo.h

```diff
@@ -14,13 +14,14 @@
 #define LLVM_CODEGEN_MACHINEBRANCHPROBABILITYINFO_H
 
 #include "llvm/CodeGen/MachineBasicBlock.h"
-#include "llvm/CodeGen/MachinePassManager.h"
 #include "llvm/Pass.h"
 #include "llvm/Support/BranchProbability.h"
 
 namespace llvm {
 
-class MachineBranchProbabilityInfo {
+class MachineBranchProbabilityInfo : public ImmutablePass {
```

Comment:
Why does the PR include all this unrelated stuff? Which part am I supposed to review? Normally I just look at the "Files changed" tab.

Reply:
The parts I actually committed. I don't know why "[martinboehme](https://github.com/martinboehme) and others added 14 commits [12 hours ago](https://github.com/llvm/llvm-project/pull/96442#commits-pushed-803aa88)" happened, adding all this other stuff to my personal branch. This has happened several other times, but not consistently

Reply:
This isn't happening for the other patches further up the stack, just this one for some reason

Reply:
I don't know how it ended up like this but I've gotten rid of these commits